src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] Args) {
+        final int ARRAY_LENGHT = 6;
+        Figure figures[] = new Figure[6];
```

Comment:
Use the constant `ARRAY_LENGTH` instead of the literal value `6` when initializing the `figures` array.